### PR TITLE
Backup and logic function customize.sh

### PR DESCRIPTION
**Pull request by tryigit**
To back up people's own settings and files just in case.

### CODE REVIEW

File: module/template/customize.sh [hunk 1]
```diff
@@ -90,18 +90,41 @@ mv "$MODPATH/libinject.so" "$MODPATH/inject"
...
+    mv "$file" "${file}.bak"
+  fi
+}
 if [ ! -d "$CONFIG_DIR" ]; then
   ui_print "- Creating configuration directory"
   mkdir -p "$CONFIG_DIR"
-  [ ! -f "$CONFIG_DIR/spoof_build_vars" ] && touch "$CONFIG_DIR/spoof_build_vars"
 fi
+
+if [ ! -f "$CONFIG_DIR/keybox.xml" ]; then
```
Comment (Young-Lord) [inline module/template/customize.sh:105]: Maybe `"$CONFIG_DIR/keybox.xml"` should be `"$CONFIG_DIR/spoof_build_vars"` here?

**Comment by Young-Lord** [comment]
Currently, if file exists, it won't be overriden. So why do we need backup?

**Comment by tryigit** [comment]
> Currently, if file exists, it won't be overriden. So why do we need backup?

It's overwriting, read the code carefully.

If you try to update the module while the keybox file exists, you will not pass Strong validation because the default file is banned. To avoid unnecessary hassle for someone who has read the Guide and added a Keybox.

**Comment by Young-Lord** [comment]
> It's overwriting, read the code carefully.

How? In [this line](https://github.com/5ec1cff/TrickyStore/pull/37/files#diff-187444b08af9e45fc3adcf427efb2801bd25428d56a8eaaec9330572f4c9a4c3R112), I can see it first detect if file exist using `[ -f filename ]`. If file doesn't exists, then execute `extract` and `mv` to create default file.

**Comment by tryigit** [comment]
> > It's overwriting, read the code carefully.
> 
> How? In [this line](https://github.com/5ec1cff/TrickyStore/pull/37/files#diff-187444b08af9e45fc3adcf427efb2801bd25428d56a8eaaec9330572f4c9a4c3R112), I can see it first detect if file exist using `[ -f filename ]`. If file doesn't exists, then execute `extract` and `mv` to create default file.

Then and else, both sections have mv command

**Comment by Young-Lord** [comment]
> Then and else, both sections have mv command

But `else` section is added by you in this PR. [Current code in master branch](https://github.com/5ec1cff/TrickyStore/blob/3a515c5fe1ce4c94d5424305afe2eaf4812a635d/module/template/customize.sh#L92-L107) doesn't include this.

**Pull request closed without merging**